src/index.ts

```diff
@@ -1,3 +1,25 @@
-export default function helloWorld(): string {
-	return 'Hello world!';
+// Imports
+import validateText from './validate-text';
+
+// Define types
+export interface W3CCSSValidatorResponse {
+	cssvalidation: {
+		validity: boolean;
+		errors?: {
+			line: number;
+			message: string;
+		}[];
+		warnings?: {
+			line: number;
+			level: 0 | 1 | 2;
+			message: string;
+		}[];
+	};
 }
```

Comment:
Does this really need to be exported from the `index.ts` file? Seems like it'd be more appropriate in the `retrieve-w3c-validation/index.ts` file, to be closer to where it's actually used.

Reply:
Maybe, I was thinking that as the package grows in the future this particular type might end up used in more places where a central file would be more useful in a central location, but if that happens we can move it at that time.